### DISCUSSION TITLE
465 GB Network Drive Limit with Mountain Duck?

### DISCUSSION THREAD

**Discussion by Jon-Badger** [iterate-ch/cyberduck#15221] [Q&A]
Hi all,

So I have 1 TB storage with iDrive-e2 and I am having issues using all of that space with Mountain Duck.   Currently the network drive is showing 73 GB free of 465 GB and I need to know how to access the other half of my online storage.  I have tried adding new buckets, server locations and bookmarks but they all report around 107 GB free of the 465 ?

[Edit] I am using Amazon S3 connection via Windows 10 Pro file explorer.

I would very much like to continue using Mountain Duck after the trial period, so if anyone can help I'd be most grateful as I'm still only half way through uploading the files I need from my PC...

Many thanks!

**Comment by tatze96** [comment]
The protocol S3 can't report quota, so we can't display the correct remaining free space. Finder and Windows Explorer will show exabyte values within the _Online_ mode and the available space within the synchronization cache location within the _Smart Synchronization_ mode for servers and protocols that don't have support for quota reporting.

**Reply by Jon-Badger** [reply]
Hi @tatze96 and thanks for your response.  I confess that I am not the smartest when it comes to this kind of thing, so I hope you will (all) forgive me when I ask...  "So does it mean I will be fine to continue using MD to upload my files past the 465Gb and continue towards the 1Tb - OR that I'm about to hit a wall that I cannot get past and will need to look for another method to continue uploading?"

**Reply by tatze96** [reply]
Within the *Smart Synchronization* mode, you'll need at least the same space as you want to upload as local space within the cache directory. Because of this, Mountain Duck will only let you transfer files until the cache location is filled. Based on your description, this would be 73 GB. After the cache location is filled, Mountain Duck will not allow you to transfer further files until you cleared some disk space.

I suggest using the *Online* mode instead of the *Smart Synchronization* mode if you want to transfer files that exceed the locally available disk space. You can switch to the *Online* mode either globally within the Mountain Duck preferences *Sync* tab or within the *Connect Mode* field in the configuration window of a specific bookmark.

**Reply by anirudhmurali** [reply]
This should be mentioned in the documentation. I was wondering the same about the drive limit. I use Smart Synchronization and I moved my cache to a different location, the disk limit is updated now 👍
